tests/base_tests.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""
+Base TestCase class
+"""
+
+import os
+import shutil
+
+from flask_testing import TestCase
+from superset.app import create_app, db
+
+superset_test_home = os.path.join(os.path.dirname(__file__), '.superset')
+shutil.rmtree(superset_test_home, ignore_errors=True)
+os.environ['SUPERSET_HOME'] = superset_test_home
+os.environ['SUPERSET_CONFIG'] = 'tests.test_config.ketchup_config'
+app = create_app()
+
+
+class SupersetTestCase(TestCase):
+    """SupersetTestCase"""
```

Comment:
Very nitpicky - feel free to ignore - but sufficiently strict Pylint checkers will complain about [PEP 257](https://www.python.org/dev/peps/pep-0257/) here.